YR/Fix-XDR-miroring-documentation/XSUP-31831 (#32128)
demisto/content: Packs/CortexXDR/Integrations/CortexXDRIR/README.md

  - The default playbook of the *Cortex XDR Incident* incident type is not *Cortex XDR Incident Sync*, change it to a 
     different playbook that does not use `XDRSyncScript`.
  - The XDR integration instance incoming mapper is set to `Cortex XDR - Incoming Mapper` and the outgoing mapper is set to `Cortex XDR - Outgoing Mapper`.

>   - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be resolved by creating a new instance with the correct mapper.

Reviewer: ```suggestion
  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be resolved by creating a new integration instance with the correct mapper. New incidents will be created and the old ones will no longer be relevant.
```

>   - Mirroring impacts only incidents fetched after successfully enabling mirroring. If incidents were fetched without mirroring or with mirroring but with the incorrect mapper, enabling mirroring with the correct mapper now won't resolve issues for those incidents. The solution is to create a new instance from scratch.

Reviewer: ```suggestion
  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be solved by resetting the last fetch run and re-fetching the incidents.
```
This should be checked, but I think that just resetting last run can be a solution instead of creating a new instance.

Author: Done.

>   - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be solved by resetting the last fetch run and re-fetching the incidents. New incidents will be created and the old ones will no longer be relevant.

Reviewer 2: ```suggestion
  - Mirroring impacts only incidents that were fetched after the mirroring was enabled for this instance. If incidents were fetched with the incorrect mapper, changing the mapper will not affect them. This can be resolved by resetting the last fetch run and re-fetching the incidents. New incidents will be created and the old ones will no longer be relevant.
```
